specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2018-01-22/iotdps.json

```diff
@@ -268,73 +268,6 @@
             "type": "string",
             "description": "This is a mandatory field, and is the logical name of the certificate that the provisioning service will access by."
           },
```

Comment:
why were these query parameters added to delete? They are not present on the PUT. Were these somehow supported on the service side?